ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -95,6 +152,14 @@ The criterion to be compared with is defined in the configuration.
         It is a match if the input value equals the defined value.
```

Comment:
[Info] The properties of an entity you can refer to are described in the relevant API documentation. --> Vielleicht allgemeinen Pfad dahin hinzufügen? *Dev Tools > API > Module name > ...*